client/state/comments/test/selectors.js

```diff
@@ -75,16 +115,27 @@ describe( 'selectors', () => {
 				1
 			);
 
-			expect( res ).to.be.eql( undefined );
+			expect( res ).toEqual( undefined );
 		} );
 	} ); // end of getPostOldestCommentDate
 
 	describe( '#getCommentLike()', () => {
-		it( 'should provide only like statistics', () => {
+		test( 'should provide only like statistics', () => {
 			const res = getCommentLike( state, 1, 1, 2 );
 
-			expect( res.i_like ).to.eql( false );
-			expect( res.like_count ).to.eql( 456 );
+			expect( res.i_like ).toEqual( false );
+			expect( res.like_count ).toEqual( 456 );
+		} );
+	} );
+
+	describe( '#getPostCommentsTree', () => {
+		test( 'should return the tree structure', () => {
+			const tree = getPostCommentsTree( state, 1, 1, 'all' );
+			expect( tree ).toMatchSnapshot();
+		} );
+
+		test( 'should reverse children', () => {
+			expect( getPostCommentsTree( stateWithDeeperChildren, 1, 1, 'all' ) ).toMatchSnapshot();
```

Comment:
it automagically knows where the snapshot is? this is so cool

Reply:
Yes, one snapshots per test file